tests/playwright-tests/utils/generateRunIdentifier.ts

```diff
@@ -0,0 +1,5 @@
+export default function generateRunIdentifier() {
+  const date = new Date();
+  const utcFormat = date.toISOString().replace('T', ' ').substring(0, 19);
+  return `UI Test${utcFormat}`;
+}
```

Comment:
The generation of this run identifier will be dependent on when the function is called. However, this isn't a good idea as this will lead to the generated run identifier being different at different times in the tests.

This means it'll be difficult to identify what things belong to the same test run (which is the point of the run identifier).

In the current Robot tests, we set up a `RUN_IDENTIFIER` environment variable once when the tests start. We don't generate this again later.

Would suggest we do something similar i.e.

```ts
// env.ts

const environment = {
  // ...
  RUN_IDENTIFIER: generateRunIdentifier(),
};
```

Then we can use this in our test cases like:

```ts
const theme = `UI Test - Test theme - ${environment.RUN_IDENTIFIER}`
```

We don't have to do this next part, but perhaps if we want to abstract this a little so that it's easier to use (and more consistent), we could create another util:

```ts
// uiTestString.ts

import environment from './environment';

export default function uiTestString(str: string): string {
  return `UI Test - ${str} - ${environment.RUN_IDENTIFIER}`;
}
```

This could then be used like:

```ts
const theme = uiTestString('Test theme'); // Creates 'UI Test - Test Theme - 2024-01-17 12:00:00'
```